cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperatorTest.java

```diff
@@ -1386,10 +1392,17 @@ public void testCruiseControlDisabled(VertxTestContext context) {
         Kafka kafka =
                 new KafkaBuilder(ResourceUtils.createKafka(CLUSTER_NAMESPACE, CLUSTER_NAME, replicas, image, healthDelay, healthTimeout))
                         .editSpec()
-                        .editKafka()
-                        .withVersion(version)
-                        .endKafka()
+                            .editKafka()
+                            .withVersion(version)
```

Comment:
Great that you are aligning this block for better readability. But you should indent this version once more.